docs/modules/ROOT/nav.adoc

```diff
@@ -1,5 +1,6 @@
 * xref:index.adoc[Getting Started]
 * xref:quickStart.adoc[Quick Start]
+* xref:create.adoc[How to Create Starter Kit]
```

Comment:
```suggestion
* xref:create.adoc[How to Create your own Starter Kit]
```

(could also have been "How to create **a** starter kit")

Reply:
I am afraid that `How to Create your own Starter Kit` is too long for nav sidebar but if you think it is okay, I am fine with it.
PS regarding skipping *a* article. It is okay to skip articles in titles and navigation menus.

Reply:
> It is okay to skip articles in titles and navigation menus

Interesting! Didn't know about that one. But given that we are not doing that in the other guides, I'd be consistent and include either "your own" or "a".